src/test/java/com/vinylteam/vinyl/web/servlets/SignUpServletTest.java

```diff
@@ -17,107 +15,56 @@
 class SignUpServletTest {
 
     private DefaultUserService mockedDefaultUserService;
-    private DefaultSecurityService mockedDefaultSecurityService;
     private HttpServletRequest mockedHttpServletRequest;
     private HttpServletResponse mockedHttpServletResponse;
-    private final User mockedUser = mock(User.class);
 
     @BeforeEach
     void beforeEach() {
-        mockedDefaultSecurityService = null;
         mockedDefaultUserService = null;
         mockedHttpServletRequest = null;
         mockedHttpServletResponse = null;
 
         mockedDefaultUserService = mock(DefaultUserService.class);
```

Comment:
Why mockedDefaultUserService = null; and then mockedDefaultUserService = mock(DefaultUserService.class);  ?
Can you write just mockedDefaultUserService = mock(DefaultUserService.class);?

Reply:
I'll try and see if verification works right.